api/spec/v1.yaml

```diff
@@ -2688,6 +2688,10 @@ components:
           type: string
         metadata_accessed:
           type: string
+        metadata:
+          description: |
+            A metadata document for this NFT instance.
+            Currently only ERC-721 is supported, where the document is an Asset Metadata from the ERC721 Metadata JSON Schema.
```

Comment:
https://eips.ethereum.org/EIPS/eip-721 their schema is here. if higher effort, we could import that and say it could be that schema